docs/content/doc/usage/backup-and-restore.en-us.md

```diff
@@ -75,7 +75,7 @@ mv custom/conf/app.ini /etc/gitea/conf/app.ini # or mv app.ini /etc/gitea/conf/a
 unzip gitea-repo.zip
 mv gitea-repo/* /var/lib/gitea/repositories/
 chown -R gitea:gitea /etc/gitea/conf/app.ini /var/lib/gitea/repositories/
-mysql -u$USER -p$PASS $DATABASE <gitea-db.sql
+mysql --default-character-set=utf8 -u$USER -p$PASS $DATABASE <gitea-db.sql
```

Comment:
should it be `utf8` or `utf8mb4` instead, as with mysql `utf8` isn't actually true `utf8`?

Reply:
The character set supplied here must be contained in the /usr/share/mysql/charsets/Index.xml file which can be found [here](https://github.com/mysql/mysql-server/blob/8.0/share/charsets/Index.xml). I only contains utf8 without mb4. The documentation about this can be found [here](https://dev.mysql.com/doc/refman/5.7/en/charset-configuration.html) for v5.7 and [here](https://dev.mysql.com/doc/refman/8.0/en/charset-configuration.html) for v8.0.

Reply:
No, you are right. It should be utf8mb4.

Reply:
I changed it now via amending.